aws_doc_sdk_examples_tools/fs.py

```diff
@@ -57,11 +57,11 @@ def read(self, path: Path) -> str:
             return file.read()
 
     def readlines(self, path: Path) -> List[str]:
-        with path.open("r") as file:
+        with path.open("r", encoding="utf-8") as file:
```

Comment:
Oh, Python3.9 and earlier. The encoding should come in the `readlines` signature, to allow the caller to provide the correct value.